app/fullService/api/buildGiftCode.ts

```diff
@@ -40,7 +37,7 @@ const buildGiftCode = async ({
   tombstoneBlock,
   valuePmob,
 }: BuildGiftCodeParams): Promise<BuildGiftCodeResult> => {
-  const { result, error }: AxiosFullServiceResponse = await axiosFullService(
+  const { result, error }: AxiosFullServiceResponse<BuildGiftCodeResponse> = await axiosFullService(
```

Comment:
similar thing here, could lift the error check from L60, include a check for `!result` and then you wouldn't have to cast. I feel like having `result` be an optional parameter and casting it like this prevents the compiler from protecting the app against a bug where we try to access `result` and it doesn't exist